DashWallet/Sources/UI/Home/Models/DWHomeModel.m

```diff
@@ -284,6 +284,12 @@ - (void)forceStartSyncingActivity {
 #pragma mark - #pragma mark - DWShortcutsModelDataSource
 
 - (BOOL)shouldShowCreateUserNameButton {
+    DSChain *chain = [DWEnvironment sharedInstance].currentChain;
+    // available in Evonet so far
+    if ([chain.devnetIdentifier isEqualToString:DWDevnetEvonetIdentifier] == NO) {
```

Comment:
chain has a parameter for isEvoEnabled (or something like) that now which is most likely better to use.